backend/app/controllers/api/v1/users/project_developers_controller.rb

```diff
@@ -0,0 +1,34 @@
+module API
+  module V1
+    module Users
+      class ProjectDevelopersController < BaseController
+        skip_before_action :require_json!, only: :create
+        before_action :authenticate_user!
+
+        def create
+          current_user.with_lock do
+            raise API::UnprocessableEntityError, I18n.t("errors.messages.user_multiple_accounts") if current_user.account_id.present?
+
+            account = Account.create! account_params
+            current_user.update! account: account, role: :project_developer
+            project_developer = ProjectDeveloper.create! project_developer_params.merge account: account
+            render json: ProjectDeveloperSerializer.new(project_developer).serializable_hash
+          end
+        end
+
+        private
+
+        def account_params
+          params.fetch(:project_developer_params, params)
```

Comment:
so the frontend will need to add root `project_developer_params`? Would that change things https://api.rubyonrails.org/classes/ActionController/ParamsWrapper.html, in base controller wrap is done for all :json requests

Reply:
Nope, that is basically optional, I have put it here just because of rswag tests, you can use:
```
{ project_developer_params: { DATA } }
or
{ DATA }
```
my guess is that FE will be using just DATA (second option)

Reply:
hmm, but maybe if i rename `project_developer_params ` to `project_developer`, I will not need this hack, will check it :)

Reply:
Soo it could theoretically work but based on documentation
```
On Active Record models with no :include or :exclude option set, it will only wrap the parameters returned by the class method attribute_names.
```
which means that I would need to manually add all translation and active storage attributes, which does not look really nice:
```
wrap_parameters include: Account.attribute_names + %i[mission location_ids about picture]
```
so I don't think it is worthy. Still, FE can use this endpoint without any root element, no need to add `project_developer_params`

Reply:
ok